packages/sanity/src/core/releases/plugin/ReleasesStudioLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import {type LayoutProps} from '../../config'
+import {BundlesProvider} from '../../store/bundles/BundlesProvider'
+import {AddonDatasetProvider} from '../../studio'
+
+export function ReleasesStudioLayout(props: LayoutProps) {
+  // TODO: Replace for useReleasesEnabled
```

Comment:
Adding this here as a placeholder if later we want to enable this only for some customers. will either remove or update after this is confirmed,